connector_flow/impexp_task.py

```diff
@@ -0,0 +1,161 @@
+        return [('file', 'File'),
+                ('chunk', 'Chunk')]
+
+    name = fields.Char(string='Name', required=True)
+    task = fields.Selection(selection='_get_available_tasks', string='Task')
+    config = fields.Text(string='Configuration')
+    last_start = fields.Datetime(string='Starting Time of the Last Run')
+    last_finish = fields.Datetime(string='Finishing Time of the '
+                                         'Last Successful Run')
+    max_retries = fields.Integer(string='Maximal Number of Re-tries'
+                                        ' If Run Asynchronously',
+                                 required=True, default=1)
+    flow_id = fields.Many2one('impexp.task.flow', string='Task Flow')
+    transitions_out_ids = fields.One2many('impexp.task.transition',
+                                          'task_from_id',
+                                          string='Outgoing Transitions')
+    transitions_in_ids = fields.One2many('impexp.task.transition',
+                                         'task_to_id',
+                                         string='Incoming Transitions')
+    flow_start = fields.Boolean(string='Start of a Task Flow')
+
+    @api.one
+    @api.constrains('flow_start', 'flow_id')
+    def _check_unique_flow_start(self):
+        """Check that there is at most one task that starts the
+           flow in a task flow"""
+        if self.flow_start:
+            flow_start_count = self.search_count(
+                [('flow_id', '=', self.flow_id.id),
+                 ('flow_start', '=', True)])
+            if 1 < flow_start_count:
+                raise exceptions. \
+                    ValidationError('The start of a task flow '
+                                    'has to be unique')
+
+    @api.multi
+    def _config(self):
+        """Parse task configuration"""
+        self.ensure_one()
+        config = self.config
+        if config:
+            return literal_eval(config)
+        return {}
+
+    @api.multi
+    def do_run(self, async=True, **kwargs):
+        self.ensure_one()
+        if async:
+            method = run_task.delay
+            kwargs.update({'description': self.name,
+                           'max_retries': self.max_retries})
+        else:
+            method = run_task
+        result = method(ConnectorSession(self.env.cr,
+                                         self.env.uid,
+                                         context=self.env.context),
+                        self._name, self.ids, async=async, **kwargs)
+        # If we run asynchronously, we ignore the result
+        #  (which is the UUID of the job in the queue).
+        if not async:
+            return result
+
+    @api.model
+    def do_run_flow(self, flow_id, **kwargs):
+        flow = self.env['impexp.task.flow'].browse(flow_id)
+        flow.ensure_one()
+        start_task = False
+        for task in flow.task_ids:
+            if task.flow_start:
+                start_task = task
```

Comment:
No you'r not. I already changed this, but it isn't detected by GH somehow. It's just the line below your comment. Please have a look at the full diff.